prep-html-links/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+
+  <a href="contact-us.html"> Click here to go to the contact page
+  </a>
```

Comment:
When you click on this link in the browser, it should load the contact page. Try an `href` attribute of `"contact-us/contact-us.html"`